cpu/cc26xx/dev/cc26xx-rf.c

```diff
@@ -1070,7 +1059,8 @@ cc26xx_rf_cpe1_isr(void)
 
```

Comment:
@tonnenpinguin 

I did actually notice this before but I left it as is because the original driver code does not re-enable interrupts on that path either. 

I've chosen not to fix this here because:
1. This PR is meant only to add an isr control interface and the changes to the driver are meant only to demonstrate this interface. I don't feel that it is appropriate to add a bug fix here.
2. The function cc26xx_rf_cpe1_isr is an interrupt handler that only runs after a low level radio error. In all my testing with this platform I've never seen this ISR run. Furthermore, the code path in question will only run if the error is completely unrecoverable. Because of the situation in which the code runs, I haven't entirely ruled out the possibility that this strange behaviour is somehow intentional (although I doubt that this is really useful behaviour in any way).

With this said, I've made a note to myself to bring this to the attention of the maintainer.